src/gui/settingsdialog.cpp

```diff
@@ -196,8 +215,17 @@ void SettingsDialog::accountAdded(AccountState *s)
 
     bool brandingSingleAccount = !Theme::instance()->multiAccount();
 
-    auto accountAction = createColorAwareAction(QLatin1String(":/client/resources/account.png"),
-                                                brandingSingleAccount ? tr("Account") : s->account()->displayName());
+    QAction *accountAction;
+    QImage avatar = s->account()->avatar();
```

Comment:
I think this will never be called since accountAdded is called before the ConnectionValidator can run?